uaclient/cli.py

```diff
@@ -1244,16 +1244,7 @@ def action_attach(args, *, cfg):
     try:
         actions.attach_with_token(cfg, token=token, allow_enable=allow_enable)
     except exceptions.UrlError:
-        msg = messages.ATTACH_FAILURE
-        event.info(msg.msg)
-        event.error(error_msg=msg.msg, error_code=msg.name)
-        event.process_events()
-        return 1
-    except exceptions.UserFacingError as exc:
-        event.info(exc.msg)
-        event.error(error_msg=exc.msg, error_code=exc.msg_code)
-        event.process_events()
-        return 1
+        raise exceptions.AttachError
```

Comment:
I think this is supposed to be instantiating the class?
```suggestion
        raise exceptions.AttachError()
```

Reply:
:facepalm: Yes, my mistake. Thanks for spotting this